### PR TITLE
Fix rn-cli linting issues

### PR DESCRIPTION
**Pull request by ignacioola**
Fixes ESLint warnings in `react-native-cli`. I isolated this PR from other lint fixes because of the top `DO NOT MODIFY THIS FILE` message. Either way I think this issues should be fixed :)

Test Plan:
----------
Run:
```
yarn lint
```
Check there are no more ESLint warnings in this file.

Release Notes:
--------------
[INTERNAL] [ENHANCEMENT] [eslint] - Fix eslint errors in react-native-cli

### CODE REVIEW

**Comment by TheSavior** [comment]
What were the lint warnings on this file? Based on you adding the underscore it seems like it is the error about unused variables which seems weird since this variable is being used throughout.

**Comment by ignacioola** [comment]
Sure, would something like `opts` be good?

**Comment by TheSavior** [comment]
You could change the top one in the file to be `args` since they come from the command line arguments:
```
var args = require('minimist')(process.argv.slice(2));
```

File: react-native-cli/index.js [hunk 1]
```diff
@@ -215,6 +214,11 @@ function createAfterConfirmation(name, options) {
   };
 
   prompt.get(property, function(err, result) {
+    if (err) {
+      console.log('Error initializing project');
+      return;
```
[the remark this answers is not in the document]
Comment (ignacioola) [reply react-native-cli/index.js]: Done!

**Pull request closed without merging**